vendor/firebase/shim.js

```diff
@@ -1,9 +1,18 @@
 /* globals Firebase */
 
-define('firebase', [], function() {
-  "use strict";
+(function() {
+  function fakeFirebase() {
+    var error = "Firebase is not available. Either the bower package is missing or your app is in Fastboot Mode.";
+    this.to = function() {
```

Comment:
The idea is to return a fake function when using Fastboot, rather than using the one that causes issues with Fastboot.